client/app/api/course/assessment/question/scribing/Scribing.js

```diff
@@ -0,0 +1,124 @@
+import { getCourseId, getAssessmentId, getScribingId } from 'lib/helpers/url-helpers';
+import BaseAPI from '../../../../Base';
+
+export default class ScribingsAPI extends BaseAPI {
+  /**
+   * scribing_question = {
+  *   id: number,
+  *   title: string,
+  *   description: string,
+  *   staff_only_comments: string,
+  *   maximum_grade: string,
+  *   weight: number,
+  *   skill_ids [],
+  *   skills: [],
+  *   published_assessment: boolean,
+  *   attempt_limit: number,
+  * }
+   */
+
+  /**
+   * Fetches a Scribing question with initial attributes
+   *
+   * @return {Promise}
+   * success response: scribing_question
+   */
+  new() {
```

Comment:
Needed this to get the assessment skills to display as skills options. The other attributes are not important but I used the same json.jbuilder template to return the attributes.